source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,105 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+1. ssh into the logging machine and use the command line.  
+**Note:** You can probably do all of this with the OpenShift UI too - https://logging-machine:8443
+
+2. Run
+
+       # $ oc edit dc logging-mux
+
+Edit the section
+
+      spec:
+        template:
+          spec:
+            containers:
+      ...
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 2Gi
+
+   For cpu, units are in millicores.  500m means 0.5 cores.
+
+   If you want mux to use more cpu, you can update it up to 1000m.
+
+   In case mux is maxed out of cpu, and you have already increased the mux cpu to 1000m, you should scale up an additional pod.
```

Comment:
Suggest:
" In the event that mux has utilized all available CPU resources, and the CPU has already been increased to 1000m, scale up to an additional pod."